lib/Page.js

```diff
@@ -115,6 +117,7 @@ class Page extends EventEmitter {
     networkManager.on(Events.NetworkManager.Response, event => this.emit(Events.Page.Response, event));
     networkManager.on(Events.NetworkManager.RequestFailed, event => this.emit(Events.Page.RequestFailed, event));
     networkManager.on(Events.NetworkManager.RequestFinished, event => this.emit(Events.Page.RequestFinished, event));
+    this._fileChooserInterceptors = new Set();
```

Comment:
Should this be a set? Does it make sense to have multiple file chooser interceptors? Maybe we should throw?

Reply:
If we were to have an event - this would behave as a set. This is done for consistency with other methods, e.g. `page.waitForRequest`.

Reply:
calling `waitForRequest` doesn't have side effects though. If multiple people are waiting for a file chooser, who calls cancel/accept?